src/snapshot/cloneplot.js

```diff
@@ -101,8 +100,11 @@ module.exports = function clonePlot(graphObj, options) {
         }
     }
 
-    var sceneIds = Plots.getSubplotIds(newLayout, 'gl3d');
-
+    // TODO: does this scene modification really belong here?
+    // If we still need it, can it move into the gl3d module?
+    var sceneIds = Object.keys(newLayout).filter(function(key) {
+        return key.match(/^scene\d*$/);
+    });
```

Comment:
`snapshot/cloneplot` will soon (I hope) be obsolete. `Plotly.toImage` does not use it anymore, only the old image server still does. All thumbnail-specific layout patches are now in `image-exporter`.